amr-wind/wind_energy/actuator/wing/wing_ops.H

```diff
@@ -164,8 +165,20 @@ struct ComputeForceOp<
             windvector[2] = (grid.vel[ip] - wdata.vel_tr) & blade_z;
 
             const auto vmag = vs::mag(windvector);
-            const auto aoa = std::atan2(windvector[2], windvector[0]) +
-                             amr_wind::utils::radians(wdata.pitch);
+
+            // Calculate the aoa based on the pitch actuation table if supplied
+            // (interpolate pitch angle at step n) or the initial pitch angle
+            amrex::Real aoa;
+            if (!wdata.pitch_timetable.empty()) {
+                const amrex::Real current_pitch = amr_wind::interp::linear(
+                    wdata.time_table, wdata.pitch_table, time.current_time());
+                amrex::Print() << "Wing Ops: Current wing pitch is "
+                               << current_pitch << " degrees.\n";
+                aoa = std::atan2(windvector[2], windvector[0]) + current_pitch;
```

Comment:
Can you modify wdata.pitch in this step and then use the same formula that's already in the code to calculate the aoa for any case?
We would not need an else statement in that case.

Reply:
Done, that is indeed more concise